src/test/testAccounts.ts

```diff
@@ -0,0 +1,159 @@
+import { ContractTransaction, ethers } from "ethers";
+import { getConfig } from "../config";
+import { register, Registration } from "../core/contracts/registry";
+import { Identity__factory, Registry__factory } from "../types/typechain";
+
+export interface RegistrationWithSigner extends Registration {
+  signer: ethers.Signer;
+}
+
+/**
+ * Use this function to set up a new signer other than what is in the config.
+ * @param accountIndex the index in the TestAccounts to get the signer for.
+ * @return the ethers.Signer associated with the test account
+ */
+export const getSignerForAccount = (accountIndex: number): ethers.Signer => {
+  if (accountIndex >= TestAccounts.length) throw new Error(`there are only ${TestAccounts.length} accounts.`);
+  const { provider } = getConfig();
+  if (!provider) throw new Error("no provider configured");
+  return new ethers.Wallet(TestAccounts[accountIndex].privateKey, provider);
+};
+
+/**
+ * Parses a DSNP Id in a contract transaction log event.
+ * @param transaction
+ * @returns the DSNP Id
+ */
+export const getIdFromRegisterTransaction = async (transaction: ContractTransaction): Promise<any> => {
```

Comment:
`Promise<any>` => `Promise<DSNPId>`?

Reply:
I added a `DSNPId` type in [my PR](https://github.com/LibertyDSNP/sdk-ts/pull/42/files#diff-4a0c39af59da784a31b673fe4bb67584304827189fc08f23feefd17bdaf4239eR6). Feel free to copy paste it and I'll just rebase when this PR gets merged in.

Reply:
Unfortunately it's not always a string.  Sometimes it's a BigNumber.  Changing it to one or the other fails tests.  This function was already written and I moved it, then added a return type to address the existing warning... which of course created a different warning.  But fixing it properly is a little more than what I felt like doing in this PR.